meta-resin-common/recipes-support/resin-expand/resin-filesystem-expand/resin-filesystem-expand

```diff
@@ -11,7 +11,16 @@ while [ ! -L "$DATA_PARTITION" ]; do
 		sleep 1 && end=$((end + 1))
 	else
 		echo "[ERROR] Timeout while waiting for filesystem label to appear."
-		return 1
+		if [ ! -f /mnt/boot/resin_attempting_fsck_recovery ]; then
```

Comment:
We introduced now a dependency on the boot partition mount service. Did you have it?

Reply:
:man_facepalming: 
I do remember a thought about the availability of `/mnt/boot`. Must've slipped into the ether.